src/demos/confirm/confirm-demo.component.html

```diff
@@ -41,5 +41,5 @@
 </div>
 
 <sky-alert *ngIf="selectedAction">
-  You selected button "{{ selectedText }}" which has an action of {{ selectedAction }}" .
+  You selected button "{{ selectedText }}" which has an action of {{ selectedAction }}."
```

Comment:
Could you also add the quotation marks before the selectedAction curly brackets? Right now they're just at the end. :)